buildpacks/nodejs-function-invoker/buildpack.toml

```diff
@@ -0,0 +1,33 @@
+api = "0.6"
+
+[buildpack]
+id = "heroku/nodejs-function-invoker"
+version = "0.1.0"
+name = "Node.js Function Invoker"
+clear-env = true
+homepage = "https://github.com/heroku/buildpacks-nodejs"
+keywords = ["nodejs", "node", "javascript", "function"]
+
+[[licenses]]
+type = "MIT"
+
+[[stacks]]
+id = "heroku-18"
+
+[[stacks]]
+id = "heroku-20"
+
+[[stacks]]
+id = "io.buildpacks.stacks.bionic"
+
+[metadata]
+
+[metadata.runtime]
+# TODO: Replace this with the actual invoker runtime archive URL.
+# This is currently using a random .tar.gz URL to test download/extraction.
+url = "https://nodejs.org/download/release/latest-v14.x/node-v14.16.1-linux-x64.tar.gz"
```

Comment:
I think we want to have some sort of validation of the downloaded file like the JVM function invoker buildpack does. We might deactivate it at first to allow quick changes in the invoker impl pre-beta, but having the feature implemented would be nice.

Reply:
I intentionally omitted that for now based on the conversation at:
https://gus.lightning.force.com/lightning/r/0D5AH0000011wd20AA/view

I'd kind of rather leave it out than have it half-commented out and not tested in this initial implementation?

Reply:
We could add it in in later PRs fairly soon after this one - I was aiming for an MVP that we could iterate on afterwards to keep the PR small - though open to ideas :-)

Reply:
That works for me too! :)